seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -471,4 +471,4 @@ optimism,openocean,2,2023-09-04,0x1c2bc7857ce9acd8d6b018eea4b2e2affbda61e5691616
 optimism,openocean,2,2023-06-30,0x4a823ca5c73264723a1326c9d3dedd0f1b2bbbc446e7f2c71ac5c998234b8ca2,9,0x1db2466d9f5e10d7090e7152b68d62703a2245f0,14.714535262567255,0x7f5c764cbc14f9669b88837ca1490cca17c31607,2
 optimism,openocean,2,2022-08-03,0x13875f629d4c608d034411333bf0f0095fb6ac41c219163ed8d359805a63fa05,25,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,11.319842,0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9,11.265711412351504
 base,maverick,1,2023-10-24,0xf76389f7730beb6a51a3e1881b0a9139fe8ec07d8403061fd52d5aaf1357ad00,12,0x50c5725949a6f0c72e6c4a641f24049a917db0cb,2955.014794642196128309,0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca,2954.059434
-zksync,maverick,1,2023-10-23,0xbe2700aac1174728513863a4f42d37aa9ac7a7d73389fa00a5abcfbd3570d5a6,4,0x2039bb4116B4EFc145Ec4f0e2eA75012D6C0f181,,0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4,
+zksync,maverick,1,2023-10-23,0xbe2700aac1174728513863a4f42d37aa9ac7a7d73389fa00a5abcfbd3570d5a6,4,0x2039bb4116B4EFc145Ec4f0e2eA75012D6C0f181,1.723727835619188444,0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4,1.7232
```

Comment:
Fixed the seed records based on this:
https://explorer.zksync.io/tx/0xbe2700aac1174728513863a4f42d37aa9ac7a7d73389fa00a5abcfbd3570d5a6